src/avk.cpp

```diff
@@ -4619,18 +4676,36 @@ namespace avk
 					"config (which is not attached to a specific color target) or assign them to specific color target attachment ids.");
 			}
 
-			// Iterate over all color target attachments and set a color blending config
-			if (result.subpass_id() >= result.mRenderPass->attachment_descriptions().size()) {
-				throw avk::runtime_error(
-					"There are fewer subpasses in the renderpass ("
-					+ std::to_string(result.mRenderPass->attachment_descriptions().size()) +
-					") than the subpass index ("
-					+ std::to_string(result.subpass_id()) +
-					") indicates. I.e. the subpass index is out of bounds.");
+			// Iterate over all color target attachments and set a color blending config 
+			size_t blending_config_num;
+			if (!dynamic_rendering_enabled)
+			{
+				const auto & renderPassVal = result.mRenderPass.value();
+				if (result.subpass_id() >= renderPassVal->attachment_descriptions().size()) {
+					throw avk::runtime_error(
+						"There are fewer subpasses in the renderpass ("
+						+ std::to_string(renderPassVal->attachment_descriptions().size()) +
+						") than the subpass index ("
+						+ std::to_string(result.subpass_id().value()) +
+						") indicates. I.e. the subpass index is out of bounds.");
+				}
+				blending_config_num = renderPassVal->color_attachments_for_subpass(result.subpass_id().value()).size(); /////////////////// TODO: (doublecheck or) FIX this section (after renderpass refactoring)
+			} 
+			// Renderpasses and Subpasses are not supported when dynamic rendering is enabled
+			// Instead we read size of the dynamic_rendering_attachments provided
+			else 
+			{
+				blending_config_num = 0;

```

Comment:
```suggestion
				blendingConfigNum = 0;

```

Please use lowerCamelCase for local variables!